rustual-boy-core/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+extern crate encoding;
+
+#[macro_use]
+extern crate nom;
```

Comment:
I don't think `nom` or `futures` are required from the core; I think only the frontend ends up using them (`futures` for some glue for CPAL and `nom` for command parsing iirc). If this is the case we should not require them here.